azext_edge/edge/providers/check/lnm.py

```diff
@@ -311,6 +313,94 @@ def get_namespace(pod: V1Pod) -> str:
     return pods
 
 
+def evaluate_scales(
+    as_list: bool = False,
+    detail_level: int = ResourceOutputDetailLevel.summary.value,
+) -> Dict[str, Any]:
+    check_manager = CheckManager(check_name="evalScales", check_desc="Evaluate LNM scales")
+
+    target_scales = "scales.autoscaling"
+    scale_namespace_conditions = ["len(scales)>=0", "spec.replicas", "status.replicas"]
+
+    scales = get_deployment_scale_grouped_by_namespace(prefix_names=[AIO_LNM_PREFIX])
```

Comment:
There is not a suitable label for this?

Reply:
not for now, I gave service team the feedback and it's tracked by https://msazure.visualstudio.com/One/_workitems/edit/25639073